tests/Extensions/Diagnostics.HealthChecks.UnitTests/HttpEndpointHealthCheckTests.cs

```diff
@@ -4,25 +4,24 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Fabric;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 using Microsoft.Extensions.Logging.Abstractions;
-using Microsoft.Omex.Extensions.Abstractions.Accessors;
-using Microsoft.Omex.Extensions.Abstractions.Activities;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
-using ServiceFabric.Mocks;
 
 namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks.UnitTests
 {
 	[TestClass]
 	public class HttpEndpointHealthCheckTests
-	{
+	{
+		internal const string PublishAddressEvnVariableName = "Fabric_NodeIPOrFQDN";
```

Comment:
Should not you use SfConfigurationProviderHelper here? do avoid duplicating constants?